disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/internal/serialization/converters/AttributesJsonConverter.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.disk.buffering.internal.serialization.converters;
+
+import com.dslplatform.json.BoolConverter;
+import com.dslplatform.json.JsonConverter;
+import com.dslplatform.json.JsonReader;
+import com.dslplatform.json.JsonWriter;
+import com.dslplatform.json.NumberConverter;
+import com.dslplatform.json.ObjectConverter;
+import com.dslplatform.json.StringConverter;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.AttributeType;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Function;
+
+@SuppressWarnings({"rawtypes", "unchecked"})
+@JsonConverter(target = Attributes.class)
+public final class AttributesJsonConverter {
```

Comment:
looks like  generated class - but doesn't have a "generated by .." comment

Reply:
I see, though it's not generated 😅 I had to create 2 custom Json converters, one for Attributes and one for generic MetricData, both in the same folder. However, these will be removed from this PR to make it smaller.